Chem | Add | Gasmasks from CBRN

MissHeda opened the pull request:
__**When merged this pull request will:**__
- Adds 1x M50 & 1x M04 gasmasks
- Edits Chemical default gasmasks 
![20230622032336_1](https://github.com/KAT-Advanced-Medical/KAM/assets/95135995/55f1306e-87e9-44be-89bb-7cfa94297d31)
![20230622032545_1](https://github.com/KAT-Advanced-Medical/KAM/assets/95135995/0002d4c5-0e87-4eea-b0e7-478128cb0f8a)

# IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

MissHeda committed:
everything working except mask texture.

MissHeda committed:
tewaks

MissHeda committed:
Merge branch 'KAT-Advanced-Medical:dev-Tomcat' into Chemical-Gasmaks

MissHeda committed:
everything working

MissHeda committed:
remove from overlays

Overlays not working due to them being projected via custom fnc in original mod.

MissHeda committed:
validate update

AtrixZockt commented on addons/chemical/XEH_preInit.sqf:
```diff
@@ -13,7 +13,7 @@ PREP_RECOMPILE_END;
     [LLSTRING(SETTING_AVAIL_GASMASK), LLSTRING(SETTING_AVAIL_GASMASK_DISC)],
     CBA_SETTINGS_CHEM,
-    "G_AirPurifyingRespirator_01_F",
+    "kat_mask_M50, kat_mask_M04",
```
Change default value to correct one. (See chemical guide)

AtrixZockt reviewed (changes requested):
Further changed need to be made once I committed something.

MissHeda committed:
CBA settings update

updated CBA settings due to PR from Digii fixing how string is handled in GasMaskList

AtrixZockt committed:
Merge branch 'dev-Tomcat' into Chemical-Gasmaks

AtrixZockt reviewed (approved):
Looks good for now
